README.md

```diff
@@ -22,6 +22,66 @@ Launch the server by typing:
 npm run dev
 ```
 
+## Local development
+
+### Ngrok setup
+
+Install ngrok globally:
+
+```bash
+npm install ngrok -g
+```
+
+Expose local server to public internet:
+
+```bash
+ngrok http 3000
+```
+
+If all goes well you should see `Tunnel States` is `online`, and Forwarding url.
+
+### Slack App setup
+
+Go to https://api.slack.com/apps?new_app=1 and create a new app.
+
+Copy Client ID, Client Secret and Signing Secret into `.env` at the root directory:
+
+```
+SLACK_CLIENT_ID=xxxxxxx
+SLACK_CLIENT_SECRET=xxxxxxx
+SLACK_CLIENT_SIGNING_SECRET=xxxxxxx
+```
+
+Click on the "Bot Users" tab, and click "Add a Bot User" button, specify a name for your bot, 
+and enable the option for "Always Show My Bot as Online", then click save.
+
+Click on the "OAuth & Permissions" tab in your Slack's app setting page, and under "Redirect URLs", add https://ngrok-url/oauth, then click save.
```

Comment:
please change https://ngrok-url/oauth to https://{ngrok-url}/oauth or https://<ngrok-url>/oauth
to indicate ngrok-url is a variable url that user will get instead of the url is "ngrok-url"